tests/unit_tests/device.cpp

```diff
@@ -130,18 +130,3 @@ TEST(device, ecdh32)
   ASSERT_EQ(tuple2.senderPk, tuple.senderPk);
 }
 
-TEST(device, ecdh8)
```

Comment:
The encode is not really an encode since the first input is now ignored as it's deterministic, so the output will not equal the input anymore after a encode/decode run.